README.md

```diff
@@ -10,7 +10,118 @@ both in CI environments and in command-line to produce a tile.
 
 ### `help`
 
+```
+$ kiln --help
+kiln
+kiln helps you build ops manager compatible tiles
+
+Usage: kiln [options] <command> [<args>]
+  --help, -h                                               bool    prints this usage information (default: false)
+  --pivotal-network-token, -pt, PIVOTAL_NETWORK_API_TOKEN  string  uaa access token for network.pivotal.io
+  --version, -v                                            bool    prints the kiln release version (default: false)
+
+Commands:
+  bake     bakes a tile
+  fetch    fetches releases
+  help     prints this usage information
+  update   updates stemcell_criteria and releases
+  version  prints the kiln release version
+```
+
+### `fetch`
+
+The `fetch` command downloads bosh release tarballs from an AWS S3 bucket to a
+a local directory specified by the `--releases-directory` flag. It discovers
+releases based on information from both the assets file and an assets lock file.
+The assets lock file name is expected to be a file in the same directory as the
+specified assets file with `lock` instead of `yaml` as the filename extension.
+The S3 object name is determined based on using regular expression capture
+groups.
+
+Kiln verifies that the checksum (SHA1) of the downloaded release matches
+checksum specified for the release in the assets lock file. If the checksums do
+not match, then the releases that don't match will be deleted from disk. *Since
```

Comment:
I am not sure about this behavior. I read the code a little and it seems like it will only delete the release if the downloaded release does not match the checksum. If a dev build is in the releases directory it will be ignored.